src/full/Agda/Interaction/Imports.hs

```diff
@@ -573,9 +573,8 @@ getStoredInterface x file isMain msi = do
 
         -- If any of the imports are newer we need to retype check
         if hs /= map snd (iImportedModules i)
-          then do
-            -- liftIO close -- Close the interface file. See above.
-            fallback
+          -- liftIO close -- Close the interface file. See above.
```

Comment:
It makes no sense to keep the commented out code if it's not in a syntactically correct position.

Reply:
Thanks for reviewing. It is removed.